test-suite/consensus-tests/src/lib.rs

```diff
@@ -1074,8 +1052,12 @@ impl SandboxBuilder {
     }
 }
 
-impl Schema<&Fork> {
+pub trait SchemaExt {
```

Comment:
IMO, this extension trait is unwarranted: the method is used in a single place, so it could be defined as `reject_transaction(&Schema, &Hash)`.